OOP excercises/1.2/Author.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * @author Gonzalo H. Mendoza
+ * 
+ */
+public class Author {
+
+	private String name;
+	private String mail;
+	private char gender;
+	
+	//Making a Constructor
+	public Author(String p_name, String p_mail, char p_gender){
```

Comment:
en los nombres de los parametros no uses p_name. Usa directamente name. Te queda this.name = name